corehq/apps/reports_core/filters.py

```diff
@@ -277,16 +277,19 @@ class ChoiceListFilter(BaseFilter):
     """
     template = 'reports_core/filters/choice_list_filter.html'
 
-    def __init__(self, name, datatype, label='Choice List Filter',
+    def __init__(self, name, field, datatype, label='Choice List Filter',
                  css_id=None, choices=None):
+        from corehq.apps.userreports.reports.filters.choice_providers import StaticChoiceProvider
         params = [
             FilterParam(name, True),
         ]
         super(ChoiceListFilter, self).__init__(name=name, params=params)
+        self.field = field
         self.datatype = datatype
         self.label = label
         self.css_id = css_id or self.name
         self.choices = choices or []
+        self.choice_provider = StaticChoiceProvider(self.choices)
```

Comment:
Taking another look, I'm a little confused what just adding the `choice_provider` and `field` properties to this existing class does; maybe I'm missing just missing it, but I don't see how this gets called/used.